src/function/scalar/string/regexp.cpp

```diff
@@ -227,9 +317,16 @@ void RegexpFun::RegisterFunction(BuiltinFunctions &set) {
 	    ScalarFunction({LogicalType::VARCHAR, LogicalType::VARCHAR, LogicalType::VARCHAR, LogicalType::VARCHAR},
 	                   LogicalType::VARCHAR, RegexReplaceFunction, false, RegexReplaceBind));
 
+	ScalarFunctionSet regexp_extract("regexp_extract");
+	regexp_extract.AddFunction(ScalarFunction({LogicalType::VARCHAR, LogicalType::VARCHAR}, LogicalType::VARCHAR,
+	                                          RegexExtractFunction, false, RegexExtractBind));
+	regexp_extract.AddFunction(ScalarFunction({LogicalType::VARCHAR, LogicalType::VARCHAR, LogicalType::ANY},
```

Comment:
Is the group supposed to be ANY type? Is it not supposed to be INTEGER?

Reply:
I used ANY because INTEGER is probably too strict. INT8/16 and UINT8/16 should be allowed too

Reply:
ANY is not required for that, implicit casts will be added from INT8 -> INT32, etc. The reason for adding the type is so that the function knows the input type it will receive, and so the implicit casts are handled centrally with a consistent set of rules (`cast_rules.cpp`).

Perhaps do BIGINT or so, so that most integer types can be promoted to it automatically?

Reply:
I've changed it to integer, see my comment other comment